coa/pkg/apis/v1alpha2/errors.go

```diff
@@ -8,6 +8,10 @@ package v1alpha2
 
 import "fmt"
 
+type ICOAError interface {
+	IsRetriableErr() bool
+}
+
```

Comment:
I didn't see anywhere we use this interface. Is that any reason for this interface?

Reply:
Renamed this interface to `IRetriableError`. It is used to check if an error has implemented the `IsRetriableErr()` function